components/carousel/__docs__/__snapshots__/storybook-stories.storyshot

```diff
@@ -63,7 +63,7 @@ exports[`DOM snapshots SLDSCarousel 3 items NoImageTest 1`] = `
             className="slds-carousel__panels slds-is-relative"
             style={
               Object {
-                "transform": "translateX(0px)",
+                "transform": "translateX(NaNpx)",
```

Comment:
`NaNpx` seems concerning.

Reply:
I believe these are the result of code within canUseDOM / canUseEventListeners conditionals not running. There are actually 56 instances of this in the snapshot, and each one is linked to numbers determined by the stageWidth DOM measurement.